content/docs/actors/pay_per_result_actors.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Actors paid per result
+description: Learn how actors that are paid per result work. Find out how this differ from the traditional "pay for usage" model and how this benefits you.
+
+menuWeight: 7.4
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actors/pay-per-result-actors
+---
+
+This page explains a new pricing model for actors in the [Apify store](https://apify.com/store) that allows you to pay only for the results an actor generates and not the underlying platform usage. This makes it transparent and easy to estimate upfront costs. If you have any feedback or would like to ask something, please join our [Discord](https://discord.gg/qkMS6pU4cF) community and let us know!
```

Comment:
This page explains how the new pricing model for actors in [Apify Store](https://apify.com/store) allows you to pay only for the results an actor generates and not the underlying platform usage.